packages/devextreme/js/events/utils/event_target.js

```diff
@@ -0,0 +1,12 @@
+export const getEventTarget = (event) => {
+    const isShadowDOMUsed = Boolean(event?.target?.shadowRoot);
+
+    if(isShadowDOMUsed) {
+        const path = event.path ?? event.composedPath?.();
+        const target = path[0];
+
+        return target;
+    }
+
+    return event?.target;
+};
```

Comment:
So, why don't u want just to put it inside `utils/index.js`? This util seems to be small and specific enough for placing it it a separate file